### PR TITLE
added update switch-name function

### PR DESCRIPTION
**Pull request by gaurav-calsoft**

### CODE REVIEW

File: ansible/library/dlink_fabric_creation.py [hunk 1]
```diff
@@ -507,12 +525,15 @@ def main():
         pn_dns_secondary_ip=dict(required=False, type='str', default=''),
         pn_domain_name=dict(required=False, type='str', default=''),
         pn_ntp_server=dict(required=False, type='str', default=''), )
-    )
+                          )
```
Comment (ashxjain) [inline ansible/library/dlink_fabric_creation.py]: Remove this extra whitespace

File: ansible/library/pn_fabric_creation.py [hunk 2]
```diff
@@ -540,7 +558,7 @@ def main():
         pn_ntp_server=dict(required=False, type='str'),
         pn_web_api=dict(type='bool', default=True),
         pn_stp=dict(required=False, type='bool', default=True), )
-    )
+                          )
```
Comment (ashxjain) [inline ansible/library/pn_fabric_creation.py]: Remove this extra whitespace

Comment (gaurav-calsoft) [inline ansible/library/pn_fabric_creation.py]: This is pylinted

File: ansible/library/dlink_fabric_creation.py (hunk 1 above)
Comment (gaurav-calsoft) [inline ansible/library/dlink_fabric_creation.py]: This is pylinted

**Pull request merged**